ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -119,10 +131,13 @@ set +e
 for i in "${!DATACENTERS[@]}"; do
   echo "$(date -u --rfc-3339=seconds) - Find virtual machines attached to ${vsphere_portgroup} in DC ${DATACENTERS[$i]} and destroy"
   DATACENTER=$(echo -n ${DATACENTERS[$i]} | tr -d '\n')
-  govc ls -json "/${DATACENTER}/network/${vsphere_portgroup}" |
-    jq '.elements[]?.Object.Vm[]?.Value' |
-    xargs -I {} --no-run-if-empty govc ls -json -L VirtualMachine:{} |
-    jq '.elements[].Path | select((contains("ova") or test("\\bci-segment-[0-9]?[0-9]?[0-9]-bastion\\b")) | not)' |
-    xargs -I {} --no-run-if-empty govc vm.destroy {}
+  portgroup_list="${vlanid} ${multizone_portgroup_1} ${multizone_portgroup_2}"
```

Comment:
I think `portgroup_list` should be defined earlier, and when dealing with `VSPHERE_MULTIZONE_LEASED_RESOURCE`, the `portgroup_list` should be updated dynamically in the loop.

Reply:
Once that, we even can handle `VSPHERE_CONNECTED_LEASED_RESOURCE` case here (sound your branch out-of-date).